packages/radix-ui-themes/src/components/alert-dialog.tsx

```diff
@@ -35,26 +35,18 @@ interface AlertDialogContentProps
 }
 const AlertDialogContent = React.forwardRef<AlertDialogContentElement, AlertDialogContentProps>(
   (props, forwardedRef) => {
-    const {
-      className,
-      forceMount,
-      container,
-      size = alertDialogContentPropDefs.size.default,
-      ...contentProps
-    } = props;
+    const { className, forceMount, container, ...contentProps } = extractProps(
+      props,
+      alertDialogContentPropDefs
+    );
     return (
       <AlertDialogPrimitive.Portal container={container} forceMount={forceMount}>
         <Theme asChild>
-          <AlertDialogPrimitive.Overlay className="rt-DialogOverlay rt-AlertDialogOverlay">
+          <AlertDialogPrimitive.Overlay className="rt-AlertDialogOverlay rt-DialogOverlay">
```

Comment:
There are a few files where classes have been reordered when there was consistency in the codebase so far:
- `AlertDialog`
- `ContextMenu`
- `DropdownMenu`
- `Link`
- `RadioGroup`